spec/models/resource_spec.rb

```diff
@@ -4,6 +4,13 @@
 describe Yt::Resource do
   subject(:resource) { Yt::Resource.new attrs }
 
+  context 'given a resource initialized with a URL (containing an ID)' do
+    let(:attrs) { {url: 'youtu.be/9bZkp7q19f0'} }
+
+    it { expect(resource.id).to eq '9bZkp7q19f0' }
```

Comment:
Won't this make an HTTP request? If so, it seems like this test belongs in the `requests` dir.

Reply:
Does it? I think it doesn't because it's a video id

Reply:
I might have just misunderstood how the url part works :)

Reply:
By the way, we might need add more tests in `requests/` directory, for this change